documentation/Installing-Clear-Containers-on-Fedora-25.md

```diff
@@ -0,0 +1,63 @@
+#Install the Clear Containers runtime on Fedora 25
+
+##Install the Clear Containers runtime
+```
+$ sudo -E dnf config-manager --add-repo
+http://download.opensuse.org/repositories/home:clearlinux:preview:clear-containers-2.1/Fedora\_25/home:clearlinux:preview:clear-containers-2.1.repo
+$ sudo dnf install cc-oci-runtime linux-container
+```
+
+##Install docker 1.12
+```
+$ sudo dnf config-manager --add-repo \
+	https://docs.docker.com/engine/installation/linux/repo_files/fedora/docker.repo
+$ sudo dnf install docker-engine-1.12.1-1.fc25
+```
+
+##Configure docker to use Clear Containers by default
+```
+$ sudo mkdir -p /etc/systemd/system/docker.service.d/
+$ sudo vi /etc/systemd/system/docker.service.d/clr-containers.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd -D --add-runtime cor=/usr/bin/cc-oci-runtime --default-runtime=cor
+```
+##Restart the docker systemd service
+```
+$ sudo systemctl daemon-reload
+$ sudo systemctl restart docker
+```
+
+##Run a Clear Container
+You are now ready to run Clear Containers. For example
+```
+$ sudo docker run -ti fedora bash
+```
+##Notes
+You can override the default runtime docker will use by specifying
+`--runtime $runtime` (`$runtime` can be either `cor` or `runc` (dockers
+default)). For example, to specify explicitly that you wish to use the
+Clear Containers runtime:
+```
+$ sudo docker run --runtime cor -ti ubuntu bash
+```
+Note: this should not be necessary if you update --default-runtime setting in clr-containers.conf as described above.
```

Comment:
probably code quote the arguments like `--default-runtime` and maybe `clr-containers.conf` as well?